addons/subscription/data/subscription_demo.xml

```diff
@@ -1,23 +1,21 @@
 <?xml version="1.0" ?>
-<openerp>
-    <data>
+<odoo>
         <record id="base.user_demo" model="res.users">
```

Comment:
Stating the obvious, it gives demo user the group "group_sale_salesman". But I am getting the purpose of this. I mean why demo user is given this group ? Can demo user perform any of the subscription activity? I guess no.
Any way, in that commit, demo user was given group "[group_tool_user](https://github.com/odoo-dev/odoo/commit/eeaa68471d8d36ae37335d78fc0ccc128aec4dcd#diff-b3883e126b8f77221d11a7b9edcb1a0aR13)", and that file has been removed from master :smile: